source/_integrations/sensehat.markdown

```diff
@@ -140,8 +140,35 @@ group:
   kitchen:
     - group.sense_hat
 ```
-
-### Directions for installing on Raspberry Pi Raspbian Based installation:
+## Installation
```

Comment:
- Headers should be surrounded by empty new lines.
- "Installation" sounds confusing right here?

Reply:
Done, instead of _Installation_ I used _Install on_